SPECS/dtc/dtc.spec

```diff
@@ -34,7 +34,7 @@ sed -i 's/SUBLEVEL = 0/SUBLEVEL = 1/' Makefile
 
 
 %build
-make %{?_smp_mflags} V=1 CC="gcc $RPM_OPT_FLAGS $RPM_LD_FLAGS"
+make %{?_smp_mflags} V=1 CC="gcc $RPM_OPT_FLAGS $RPM_LD_FLAGS -w"
```

Comment:
In general treating warnings as errors is good practice and I'd prefer not to do globally disable that for this package. Before we jump into it, do you know what changed between now and when the last time the package build successfully? We've had no issues on the 7<sup>th</sup> of April for instance.

Also, if it's going to turn out that this regression is expected, then I'd at least only limit this change to ignoring `cast-qual` and `missing-prototypes` warnings, so that we can still catch any regressions in the future.